clang/lib/Driver/Driver.cpp

```diff
@@ -5855,7 +5855,7 @@ class OffloadingActionBuilder final {
         LibraryPaths.emplace_back(WithInstallPath.c_str());
 
         // Select libclc variant based on target triple
-        std::string LibSpirvTargetName = "builtins.link.libspirv-";
+        std::string LibSpirvTargetName = "libspirv-";
```

Comment:
Is there a way to verify this in a driver test?

Reply:
I think it is a bit tricky, tests in `clang/Driver` all seem to use the `-fsycl-libspirv-path` option, probably because the presence of that library is not an hard requirement for compilation to succeed (the driver in general doesn't seem to error out if it can't find it), and I'm not sure if it is safe to assume that `libclc` is built and can be found in the right path when lit tests for `clang` are run (in CMake terms, `libclc` depends on `clang`, but `clang` doesn't depend on `libclc`). We have a check for `llvm-link` action for `libclc` in [out driver lit test](https://github.com/intel/llvm/blob/sycl/clang/test/Driver/sycl-native-cpu-fsycl.cpp), but as I mentioned it uses `-fsycl-libspirv-path`.

Reply:
One of the existing [driver tests for cuda](https://github.com/intel/llvm/blob/sycl/clang/test/Driver/cuda-nvptx-target.cpp#L7) seems to check for the installed `libspirv` (without `-fsycl-libspirv-path`), so nativecpu could probably do it in a similar way?

Reply:
Thanks, I think that the issue with this is that we need to merge https://github.com/intel/llvm/pull/13052 first, which will enable the native cpu  backend in github CI, making it so the `libclc` file we need for native cpu is built, and the driver will be able to find it. I've temporarily added the test in `sycl/test/check_device_code/native_cpu`, so that we can use the `REQUIRES: native_cpu` directive, and I'll move the test once the CI PR gets merged (incidentally, the CI PR is blocked by this one). @mdtoguchi how does that sound?

Reply:
Sounds reasonable to me